DEPLOY.md

```diff
@@ -1,6 +1,6 @@
 # Deployment
 
-When this application is deployed you will need to do some setup. These instructions are for an Ubuntu server running an `nginx` reverse proxy  and `gunicorn`. 
+When this application is deployed you will need to do some setup. These instructions are for an Ubuntu server running an `nginx` reverse proxy  and `gunicorn`.
```

Comment:
My text editor apparently removed that extra space. Happy to remove this line change if you prefer.